mix.exs

```diff
@@ -25,11 +25,12 @@ defmodule SortedSet.MixProject do
   defp deps do
     [
       {:rustler, "~> 0.22.0"},
+      {:jemalloc_info, "~> 0.1", app: false},
       {:ex_doc, "~> 0.19", only: [:dev], runtime: false},
       {:benchee, "~> 1.0", only: [:dev]},
       {:benchee_html, "~> 1.0", only: [:dev]},
       {:stream_data, "~> 0.4", only: [:test]},
-      #{:dialyxir, "~> 1.0.0-rc.3", only: [:dev], runtime: false}
+      {:dialyxir, "~> 1.0.0-rc.3", only: [:dev], runtime: false}
```

Comment:
We can probably go ahead and upgrade to latest (1.1.0), no need to specify an rc anymore.